### PR TITLE
API-1370: API reference cheat sheet for product model updated event

### PR DESCRIPTION
**Pull request by tomglvng**
add API reference cheat sheet for product model updated event

### CODE REVIEW

**Review by pierrejolly — APPROVED** [review]

**Review by fitn — APPROVED** [review]

File: content/events-reference/events-reference-serenity/resources/product-model-updated.yml [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+              description: Product model code
+            enabled:
+              type: boolean
+              description: Whether the product model is enable
+            family:
+              type: string
+              description: Family code from which the product model inherits its attributes and attributes requirements
+            family_variant:
+              type: string
+              description: Family vairant code from which the product model inherits its attributes and attributes requirements
```
Comment (meganelepalud-akeneo) [inline content/events-reference/events-reference-serenity/resources/product-model-updated.yml:40]:
```suggestion
              description: Family variant code from which the product model inherits its attributes and attribute requirements
```

File: content/events-reference/events-reference-serenity/resources/product-model-updated.yml [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+          properties:
+            code:
+              type: string
+              description: Product model code
+            enabled:
+              type: boolean
+              description: Whether the product model is enable
+            family:
+              type: string
+              description: Family code from which the product model inherits its attributes and attributes requirements
```
Comment (meganelepalud-akeneo) [inline content/events-reference/events-reference-serenity/resources/product-model-updated.yml:37]:
```suggestion
              description: Family code from which the product model inherits its attributes and attribute requirements
```

File: content/events-reference/events-reference-serenity/resources/product-model-updated.yml [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+            family_variant:
+              type: string
+              description: Family vairant code from which the product model inherits its attributes and attributes requirements
+            parent:
+              type: string
+              description: Code of the parent product model when the product is a variant
```
Comment (meganelepalud-akeneo) [inline content/events-reference/events-reference-serenity/resources/product-model-updated.yml:43]:
```suggestion
              description: Code of the parent product model when this a level two product model
```

File: content/events-reference/events-reference-serenity/resources/product-model-updated.yml [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+              type: string
+              description: Date of the last product update
+              format: dateTime
+            metadata:
+              type: object
+              description: More information around the product model (only available in the Enterprise Edition)
+              properties:
+                workflow_status:
+                  description: Status of the product model regarding the user permissions (only available in the Enterprise Edition)
+                  type: string
```
Comment (meganelepalud-akeneo) [inline content/events-reference/events-reference-serenity/resources/product-model-updated.yml:127]: (+) saut de ligne?

File: content/events-reference/events-reference-v5/product-models-50.md [hunk 5]
```diff
@@ -5,7 +5,122 @@ TODO
 TODO
 
 ## Product model udpated
-TODO
+
+### Format
+A product model updated event follows this format:
+::: event_api_reference content/events-reference/events-reference-serenity/resources/product-model-updated.yml
```
Comment (meganelepalud-akeneo) [inline content/events-reference/events-reference-v5/product-models-50.md:11]: same comment as for the product model created event type.

**Review by meganelepalud-akeneo — CHANGES_REQUESTED** [review]

**Pull request merged**